docs/reference/commandline/ps.md

```diff
@@ -24,6 +24,7 @@ weight=1
       -q, --quiet=false     Only display numeric IDs
       -s, --size=false      Display total file sizes
       --since=""            Show created since Id or Name, include non-running
+      -F, --format=[]       Pretty-print containers using a Go template
```

Comment:
If we merge this we'll have:
```
docker inspect -f/--format
docker ps -f/--filter  -F/--format
docker version -f/--format
```
Since it easier to add more options later, perhaps removing `-F` for now might be the safest.
I just don't know if having `-f` and `-F` is asking for confusion and typos. If we had a ton of options and ran out of letters it seems reasonable, but given the small number of flags we have its a shame we ran into this.